app/drivers/zephyr/dts/bindings/sensor/alps,ec11.yaml

```diff
@@ -19,3 +19,7 @@ properties:
     type: int
     description: Number of pulses per tick
     required: false
+  steps:
```

Comment:
Should we call this `steps-per-rotation` to match `triggers-per-rotation`?

Reply:
The `steps` naming was chosen to match the NRFX qdec driver that uses the same property name.